backend/db/initialize.py

```diff
@@ -59,6 +59,7 @@
     "table" text not null,
     primary_key text not null,
     status_initially text not null,
+    error_datetime timestamp with time zone,
```

Comment:
## Add: `error_datetime`
Added field. Not using in codebase currently; just there for troubleshooting purposes / data collection / possible future use. Added via:
```sql
ALTER TABLE your_table_name
ADD COLUMN error_datetime TIMESTAMP WITH TIME ZONE DEFAULT current_timestamp;
```